qa/vagrant/src/test/resources/packaging/scripts/plugin_test_cases.bash

```diff
@@ -197,6 +197,15 @@ fi
     assert_file_exist "$ESHOME/plugins/site-example/_site/index.html"
 }
 
+@test "[$GROUP] check the installed plugins can be listed with 'plugins list' and result matches the list of plugins in plugins pom" {
+    list_plugins | tail -n +2 | sed 's/^......//' | \
+        sort > /tmp/installed
+    ls /elasticsearch/plugins/*/pom.xml | cut -d '/' -f 4 |
```

Comment:
Maybe this comparison should be pulled into a function though because we do it twice. Maybe. I dunno. Its a bit fiddly and I don't like copying and pasting it.